System/Desk/DIButton.ts

```diff
@@ -44,14 +44,16 @@ export class DIButton extends DIView {
         this.buttonBody.textContent = value;
     }
 
+    // eslint-disable-next-line class-methods-use-this
     get width() {
         // @ts-ignore TODO: bug - maybe use this._width?
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
         return _width;
     }
 
-    set width(value) {
+    set width(value: number) {
         this._width = value;
-        this.buttonBody.style.width = "".concat(value, "px");
+        this.buttonBody.style.width = "".concat(`${value}`, "px");
```

Comment:
Is '${value}' preferred over value.toString()?

Reply:
It's my preference, but I don't have a reason

Reply:
Updated